build/Targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
-
+  
```

Comment:
Yes, but only when the formatting engine runs (`;`, `}` or FormatDocument) - the proposed feature for all languages is to do it on save if the .editorconfig property is set...